src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
+import mate.academy.hibernate.relations.exeptions.DataProcessingException;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDaoImpl actorDao;
```

Comment:
It is a common mistake to directly instantiate an implementation class like ActorDaoImpl. Instead, use dependency injection and program to an interface, not an implementation. Consider changing the field type to the interface ActorDao and inject the implementation.